cypress/support/commands.js

```diff
@@ -0,0 +1,54 @@
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+//
+//
+// -- This is a parent command --
+// Cypress.Commands.add('login', (email, password) => { ... })
+//
+//
+// -- This is a child command --
+// Cypress.Commands.add('drag', { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add('dismiss', { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+import 'cypress-file-upload';
+import { loginSelectors, profileImageUploadSelector } from '../constants/selectors/selector'
+import { texts } from '../constants/texts/text'
+
+Cypress.Commands.add('login',(email, password) => {
+    cy.get(loginSelectors.emailField).type(email);
+    cy.get(loginSelectors.passwordField).type(password);
+    cy.get(loginSelectors.loginSubmit).click();
+})
+
+Cypress.Commands.add('msgPrompt',(msg) => {
+    cy.get(profileImageUploadSelector.toasterMessageContainer).should('have.text', msg);
+})
+
+Cypress.Commands.add('uploadImg',(imgName, message) => {
+    cy.get(profileImageUploadSelector.uploadImg).attachFile(imgName);
+    cy.get(profileImageUploadSelector.toasterMessageContainer).should('have.text', message);
+})
+
+Cypress.Commands.add('changeImg',(imgName, message) => {
+    cy.get(profileImageUploadSelector.changeImg).attachFile(imgName);
+    cy.get(profileImageUploadSelector.toasterMessageContainer).should('have.text', message);
+})
+
+
+Cypress.Commands.add('loginSuccessAssert', () => {
```

Comment:
If the code is reused in almost every test suite, then we can create a custom command for it. But if the code is used in only some of the test suites, we can create a utility function and import it whenever needed.